src/components/Clubs/SmartTagSuggester.tsx

```
@@ -0,0 +1,178 @@
+
+    debounceTimer.current = setTimeout(async () => {
+      setLoading(true);
+      try {
+        const { data, error } = await supabase.functions.invoke("suggest-club-tags", {
+          body: { text: missionText },
+        });
+
+        if (!error && data?.tags) {
+          // Filter out tags that are already selected
+          const filtered = (data.tags as string[]).filter(
+            (tag) => !selectedTags.some((selected) => selected.toLowerCase() === tag.toLowerCase())
```

ESLint

[error] Insert `,`

(prettier/prettier)